apps/sms/js/activity_handler.js

```diff
@@ -294,70 +293,56 @@ var ActivityHandler = {
       };
 
     if (Compose.isEmpty()) {
-      this.launchComposer(activity);
-    } else {
-      // ask user how should we do
-      ActivityHandler.displayUnsentConfirmation(activity);
+      return this.launchComposer(activity);
```

Comment:
I didn't like it earlier, but now I like early return :) So that I can stop reading the function for the use case I was following once I see return and can be sure that the rest of function is for another use case.